gpjax/types.py

```diff
@@ -23,6 +23,13 @@ def __repr__(self) -> str:
             f"- Number of datapoints: {self.X.shape[0]}\n- Dimension: {self.X.shape[1]}"
         )
 
+    def __add__(self, other: "Dataset") -> "Dataset":
+        """Combines two datasets into one."""
```

Comment:
Can we just be explicit here that the right-hand dataset is stack beneath the left-hand dataset?

Reply:
Good idea. Will expand the description.